models/edr/alerts/error_models.sql

```diff
@@ -0,0 +1,23 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+select model_execution_id as alert_id,
```

Comment:
The dbt way is to define a cte with 'with' statement to select the entire model_run_results, then the next CTE reads it and filters the result and the last one simply select everything (see alerts_dbt_tests for an example). This will make it easier for our users to read it